source/reference/supported-server-version.txt

```diff
@@ -17,23 +17,50 @@ version limitations and requirements:
 
 - .. include:: /includes/fact-no-8.0-support.rst
 
-- .. include:: /includes/fact-minimum-server-version-support.rst
-
 - ``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
 
-- .. include:: /includes/fact-minimum-fcv.rst 
+- Your source cluster must run one of the following major supported versions: 
+  
+  - 4.4 (*New in version 1.10.0*)
+  - 5.0 (*New in version 1.10.0*)
+  - 6.0
+  - 7.0
+
+The following table lists the minimum MongoDB server version requirements for  
+source clusters:
```

Comment:
```suggestion
clusters:
```

Technically mongosync also enforces these min patch requirements on the destination cluster.

Reply:
noted!